src/main/webapp/app/entities/book/book.component.ts

```diff
@@ -29,6 +30,7 @@ export class BookComponent implements OnInit, OnDestroy {
     predicate: any;
     previousPage: any;
     reverse: any;
+    private _filterOnlyCheapBooks = false;
```

Comment:
regarding naming convention, I think we usually don't start with `_`

Reply:
Are you sure? 
I think it's a common convention to add it if you have setters&getters

Reply:
no, not sure - please check with Anton